src/App.tsx

```diff
@@ -1,6 +1,8 @@
 import { Routes, Route, BrowserRouter as Router } from 'react-router-dom';
 import './index.css';
 //pages
+import About from './components/pages/About/About';
+import FAQ from './components/pages/FAQ/FAQ';
 import Home from './components/pages/Home/Home';
 //layout
 import NavBar from './components/layout/Navbar/Navbar';
```

Comment:
we can create `index.ts` in `/components/pages` that exports all of our pages from there, so we are then able to import pages a bit more concisely

e.g.
```
import { About, FAQ, Home } from './components/pages'`
```

Reply:
requested changes added in commit a725b9f156d13fb7fc917371c2ff64d339103f87